### PR TITLE
Reference the correct command: insert -> delete

### PR DESCRIPTION
**Pull request by NiceGuyIT**
# Description
The docs reference "insert into" for the "delete" command.

# User-Facing Changes
N/A

# Tests + Formatting
I don't know of any tests for docs.

# After Submitting
N/A

### CODE REVIEW

**Comment by ysthakur** [comment]
Since `stor delete` can also delete tables, perhaps "the table you want to delete/delete from" would be better than just "delete from"?

**Comment by NiceGuyIT** [comment]
@ysthakur Good point. PR has been updated.

File: crates/nu-command/src/stor/delete.rs [hunk 1]
```diff
@@ -16,7 +16,7 @@ impl Command for StorDelete {
             .required_named(
                 "table-name",
                 SyntaxShape::String,
-                "name of the table you want to insert into",
+                "name of the table you want to delete/delete from",
```
Comment (fdncred) [inline crates/nu-command/src/stor/delete.rs:19]:
```suggestion
                "name of the table you want to delete or delete from",
```

**Review by fdncred — APPROVED** [review]

**Comment by fdncred** [comment]
Thanks!

**Pull request merged**